src/Riok.Mapperly/Descriptors/MappingBodyBuilders/ObjectMemberMappingBodyBuilder.cs

```diff
@@ -66,6 +66,20 @@ public static void BuildMappingBody(IMembersContainerBuilderContext<IMemberAssig
             }
         }
 
+        foreach (var targetMember in ctx.IgnoredTargetMembers.Values)
+        {
+            if (ctx.MemberConfigsByRootTargetName.Remove(targetMember.Name, out var memberConfigs))
+            {
+                // add all configured mappings
```

Comment:
probably add ` even if they are ignored` and move it up to L69 as this is where the section begins IMO.